setup.py

```diff
@@ -4,7 +4,7 @@
 """
 
 from __future__ import print_function, absolute_import
-
+from string import Template
```

Comment:
Weird that I've never used this, but it seems like the right tool for the job...